tests/suites/1_stateful/00_copy/00_0009_copy_csv_with_computed_column.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+# Should be <root>/tests/data/
+DATADIR=$(realpath $CURDIR/../../../data/)
+
+echo "drop table if exists sample" | $MYSQL_CLIENT_CONNECT
+
+## Create table
+cat <<EOF | $MYSQL_CLIENT_CONNECT
+CREATE TABLE sample
```

Comment:
does it work if virtual columns are not the last ones in schema? 

```
(
    Id     INT,
    City2  VARCHAR AS (reverse(City)) STORED,
    Score2 INT64 AS (Score + 2) VIRTUAL,
    City   VARCHAR,
    Score  INT,
);
```

Reply:
yes